Geometry/HcalTowerAlgo/test/runHcalGeometryPlan1Tester_cfg.py

```diff
@@ -0,0 +1,17 @@
+import FWCore.ParameterSet.Config as cms
+
+process = cms.Process("HcalGeometryTest")
+
+process.load("Geometry.HcalCommonData.testGeometry17bXML_cfi")
```

Comment:
@bsunanda - here is where the geometry choice is made and it is from XML, not DB